init.lua

```diff
@@ -2469,18 +2511,17 @@ command.add(
   end,
 
   ["lsp:rename-symbol"] = function(doc)
-    local symbol = doc:get_text(doc:get_selection())
-    local line1, col1, line2 = doc:get_selection()
-    if #symbol > 0 and line1 == line2 then
-      core.command_view:enter("New Symbol Name", {
-        text = symbol,
-        submit = function(new_name)
-          lsp.request_symbol_rename(doc, line1, col1, new_name)
-        end
-      })
-    else
-      core.log("Please select a symbol on the document to rename.")
-    end
+      local symbol, line1, col1, line2, _ = get_current_symbol_info(doc)
+      if #symbol > 0 and line1 == line2 then
```

Comment:
```suggestion
      if symbol and #symbol > 0 and line1 == line2 then
```
because `get_current_symbol_info` can return `nil`.